.github/workflows/ciwheel.yml

```diff
@@ -0,0 +1,91 @@
+name: Arbor on Wheels
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build_binary_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: recursive
+
+      - name: Build wheels Linux
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        uses: joerick/cibuildwheel@v1.9.0
+        with:
+          output-dir: dist
+        env:
+          CIBW_BEFORE_BUILD: python -m pip install numpy setuptools
+          CIBW_BUILD: "cp3?-manylinux_x86_64"
+          CIBW_SKIP: "cp35-*"
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+          # CIBW_TEST_COMMAND: TODO
+
+      - name: Build wheels macos
+        if: ${{ startsWith(matrix.os, 'macos') }}
+        uses: joerick/cibuildwheel@v1.9.0
+        with:
+          output-dir: dist
+        env:
+          MACOSX_DEPLOYMENT_TARGET: "10.15" #needed to undo some CIBW settings
+          CIBW_BEFORE_BUILD: python -m pip install numpy setuptools
```

Comment:
This (and CIBW_SKIP) can be set once on the job level env

Reply:
In fact, it's probably cleaner to use not the cartesian product matrix, but instead the config variant to set these variables (see basic.yml for how this looks like). The macos specifics can be set always (or, if we are worried about pollution using `run ::set ENV=VAR`)

Reply:
The Action runs this step in a docker image in the case of Linux (the manylinux2014 image) that does not inherit those things. It's clearer that this step is 'special' by keeping it contained like this.

Come to think of it, I probably don't need to setup Python.

Reply:
Seems weird that the same keyword has different scope, but well.